### PR TITLE
Create shelf, swiper and grid components for cards and tiles

### PR DESCRIPTION
**Pull request by thomasthesecond**
I might need to re-write the swiper to use react-slick. I need to see if provides me the same functionality.

### CODE REVIEW

File: src/components/cardShelfVideoSwiper/index.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    width: "calc(100% + 100px)",
+    padding: "30px 50px 80px",
+    marginTop: "-30px",
+    marginLeft: "-50px",
+    overflow: "hidden",
+    position: "relative",
+  },
+};
+
+const swiperOptions = {
```
Comment (nickfranciosi) [inline src/components/cardShelfVideoSwiper/index.jsx]: @thomasthesecond  does this library have responsive options. Looking at the comp it appears the arrows disappear.

File: src/components/cardShelf/index.jsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export CardShelf from "./cardShelf";
+export CardShelfContent from "./cardShelfContent";
+export CardShelfHeader from "./cardShelfHeader";
+
+export default from "./cardShelf";
```
Comment (nickfranciosi) [inline src/components/cardShelf/index.jsx]: Not sure CardShelf needs to be exported as both a named export and default

File: src/components/cardShelfVideo/index.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+);
+
+CardShelfVideo.propTypes = {
+  children: (props, propName, componentName) => {
+    const prop = props[propName];
+    let error = null;
+
+    React.Children.forEach(prop, (child) => {
+      if (child.type !== CardVideo) {
+        error = new Error(`${componentName} children should be of type "CardVideo".`);
```
Comment (nickfranciosi) [inline src/components/cardShelfVideo/index.jsx]: Will that taller card we need to build also be of type CardVideo? We may need to loosen this up a bit

File: src/components/cardShelfVideoSwiper/index.jsx (hunk 1 above)
Comment (thomasthesecond) [inline src/components/cardShelfVideoSwiper/index.jsx]: It does

File: src/components/cardShelf/index.jsx (hunk 2 above)
Comment (thomasthesecond) [inline src/components/cardShelf/index.jsx]: Will remove

**Pull request merged**